src/javaharness/java/arcs/crdt/VersionMap.java

```diff
@@ -0,0 +1,5 @@
+package arcs.crdt;
+
+import java.util.HashMap;
+
+public class VersionMap extends HashMap<String, Integer> {}
```

Comment:
Does this need to be public, is package-protected (rm public) sufficient? Same for all other `public` interfaces/classes in this PR. I think only a small number of classes actually need to be public.

Reply:
currently i'm using them all in the upcoming pr in StorageProxy and CollectionProxy classes, that are in arcs.api package. 
it will probably make sense to do more refactoring and move that to arcs.storage package, but i'd rather do it after the implementation is functional. 
but also that won't address the public issue - these will still be separate packages, so i'm not sure what's the best way to address this.